meinberlin/apps/contrib/static/js/unload_warning.js

```diff
@@ -0,0 +1,24 @@
+/* global $ CKEDITOR */
+
+$(function () {
+  var submitted = false
+  var changeHandler = function () {
+    $(window).on('beforeunload', function () {
+      if (!submitted) {
+        return false
+      }
+    })
+  }
```

Comment:
The spec says that you should set [`event.returnValue`](https://www.w3.org/TR/html5/browsers.html#dom-beforeunloadevent-returnvalue) to a message. [MDN](https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload) says that some browsers ignore the message and always show a default one. I think it does not hurt to add some compatibility code here.